javascript.js

```diff
@@ -1,3 +1,163 @@
+//Here is where the html are created with js for the popup window
+var fstElemDiv = document.createElement('div');
+fstElemDiv.classList.add('container_pop');
+document.getElementById('pop_window').appendChild(fstElemDiv);
+
+var sndElemDiv = document.createElement('div');
+sndElemDiv.classList.add('container_header');
+document.getElementsByClassName('container_pop')[0].appendChild(sndElemDiv);
+
+var fstElemPar = document.createElement('p');
+fstElemPar.classList.add('title_card_one');
+document.getElementsByClassName('container_header')[0].appendChild(fstElemPar);
+
+var trdElemDiv = document.createElement('div');
+trdElemDiv.classList.add('disable_x');
+trdElemDiv.setAttribute("onclick","show_normal();");
+document.getElementsByClassName('container_header')[0].appendChild(trdElemDiv);
+
+var frthElemDiv = document.createElement('div');
+frthElemDiv.classList.add('card_one');
+document.getElementsByClassName('container_pop')[0].appendChild(frthElemDiv);
+
+var sndElemPar = document.createElement('p');
```

Comment:
[OPTIONAL] While you did a good job trying to dynamically display your objects 👍 , I think it would be better if you write actual HTML code in your js file wrapped in a string literal. This will make your code shorter, cleaner, and easy to maintain. 

You can see that you have written so many lines of code to create HTML elements which is not making your code base clean and readable. Here is an example 👇 

```
document.querySelector('.works').innerHTML = projectDatas
  .map(
    (data) => `
    <section class="project">
<div class="project-img">
<img
src= ${data.featuredImage}
class="snapshoot"
alt="Tonic project image"
/>
</div>
<div class="project-description">
<h2 class="project-title">${data.name}</h2>
<ul class="project-author">
<li>${data.author}</li>
<li>${data.stack}</li>
<li>${data.year}</li>
</ul>
<p class="project-text"
>${data.description}</p
>
<ul class="project-tools">
<li>${data.technologies[0]}</li>
<li>${data.technologies[1]}</li>
<li>${data.technologies[2]}</li>
</ul>
<button type="button" class="btn">See Project</button>
</div>
</section>
    `,
  )
  .join('');

```

Reply:
### Ok now is everything in one array.

![image](https://user-images.githubusercontent.com/70304755/180071103-709fa073-2694-4d02-adb9-f7d8daff24c8.png)